packages/third-parties/src/google/gtm.tsx

```diff
@@ -31,6 +31,15 @@ export function GoogleTagManager(props: GTMParams) {
   const gtmAuth = auth ? `&gtm_auth=${auth}` : ''
   const gtmPreview = preview ? `&gtm_preview=${preview}&gtm_cookies_win=x` : ''
 
+  useEffect(() => {
+    // Useful for feature detection and measurement
```

Comment:
```suggestion
    // performance.mark is being used as a feature use signal. While it is traditionally used for performance
    // benchmarking it is low overhead and thus considered safe to use in production and it is a widely available
    // existing API.
```

Reply:
maybe update both comments. I interpretted feature detection as being related to whether the enviornment provides a feature and measurement as this being related to preformance measurement given the APIs used. Clarifying should help us keep track of why this code is in here when we rediscover this later :)